test.html

```diff
@@ -1,162 +1,207 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Bribbble</title>
-    <link rel="stylesheet" href="test.css">
-</head>
-<body>
-    <header class="header">
-        <nav>
-            <a href="bribbble.html"><img id="logo" src="Images\logo.jpg"></a>
-            <div class="menu" id="menuLink">
-                <ul class="ul1">
-                    <li><a href="" class="nav_btn">Inspiration</a></li>
-                    <li><a href="" class="nav_btn">Find work</a></li>
-                    <li><a href="" class="nav_btn">Learn Design</a></li>
-                    <li><a href="" class="nav_btn">Go Pro</Prog></a></li>
-                    <li><a href="" class="nav_btn">Marketplace</a></li>
-                    <li><a href="" class="nav_btn">Hire Designers</a></li>
-                </ul>
-                <i class="fa-solid fa-xmark" onclick="menuoff()"></i>
-            </div>
-            <i class="fa-solid fa-bars" onclick="menuon()"></i>
-            <div class="sign_menu">
-               <ul class="ul2">
-                    <li><a href=""><i class="fa-regular fa-magnifying-glass"></i></a></li>
-                    <li><a href="">Sign in</a></li> /
-                    <li><a href="">Sign up</a></li>
-                    <li><a href="" class="free_btn">Start a free project</a></li>
-                </ul>
-            </div>
-        </nav>
-        <hr>
+    <link rel="stylesheet" href="test.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="bribbble.html"><img id="logo" src="images/logo.jpg" /></a>
+        <div class="menu" id="menuLink">
+          <ul class="ul1">
+            <li><a href="" class="nav_btn">Inspiration</a></li>
+            <li><a href="" class="nav_btn">Find work</a></li>
+            <li><a href="" class="nav_btn">Learn Design</a></li>
+            <li><a href="" class="nav_btn">Go Pro</a></li>
+            <li><a href="" class="nav_btn">Marketplace</a></li>
+            <li><a href="" class="nav_btn">Hire Designers</a></li>
+          </ul>
+          <i class="fa-solid fa-xmark" id="close-menu"></i>
+        </div>
+        <i class="fa-solid fa-bars" id="open-menu"></i>
+        <div class="sign_menu">
+          <ul class="ul2">
+            <li>
+              <a href=""><i class="fa-solid fa-magnifying-glass"></i></a>
+            </li>
+            <li><a href="">Sign in</a></li>
+            /
+            <li><a href="">Sign up</a></li>
+            <li><a href="" class="free_btn">Start a free project</a></li>
+          </ul>
+        </div>
+      </nav>
+      <hr />
     </header>
-    <br><br><br>
```

Comment:
Do not use <br>. Instead use positional CSS to ensure page1 is pushed down by a number of pixels.